elpis/gui/src/components/Model/DownloadButton.js

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from "react";
+import {Button, Loader, Segment} from "semantic-ui-react";
+import urls from "urls";
+
+export default function DownloadButton() {
+  const [downloading, setDownloading] = useState(false);
+  const download_model = async () => {
+    setDownloading(true);
+    await fetch(urls.api.model.download);
+    setDownloading(false);
+  };
+
+  return (
+      <Segment>
+          <Button onClick={download_model}>Download Model</Button>
+          {downloading && (
+              <Loader indeterminate active>
+                  {/* TODO Add size of file being downloaded */}
+                  Zipping and Downloading Model. This may take a long time.
```

Comment:
We don't want to scare the children (users).
```suggestion
                  Zipping and Downloading Model. This may take a some time.
```

Reply:
Typo, also remove the `a` : "This may take some time."

Reply:
Good catch, thanks :)